pkgs/by-name/ll/llhttp/package.nix

```diff
@@ -28,9 +29,12 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   cmakeFlags = [
-    "-DBUILD_STATIC_LIBS=ON"
+    "-DBUILD_SHARED_LIBS=ON"
```

Comment:
Looks like this change breaks restinio: ([full log](https://paste.fliegendewurst.eu/d0yb0c.log))

```
CMake Error at restinio/CMakeLists.txt:107 (message):
  Unable to identify how to depend on llhttp.  Giving up...
-- Configuring incomplete, errors occurred!
```